customer.go

```diff
@@ -2,6 +2,8 @@ package goshopify
 
 import (
 	"fmt"
+	"github.com/shopspring/decimal"
```

Comment:
Minor: Could you separate this import from the standard library imports? Like [this](https://github.com/getconversio/go-shopify/blob/f33730dd1d730e846c20adaa93ac4b84fa315c8c/goshopify.go#L14-L16)